neutrino-custom/base.js

```diff
@@ -83,7 +82,7 @@ module.exports = neutrino => {
         .end();
     neutrino.config
         .entry('userguide')
-        .add(path.join(UI, 'entry-userguide.js'))
+        .add(path.join(UI, 'userguide-view', 'index.jsx'))
```

Comment:
I think perhaps we should call the directory name `userguide` rather than `userguide-view`?

(The `job-view` and `test-view` names are because they are two different ways of viewing the CI data, and `ui/job/` wouldn't have been clear enough -- but for other pages that have clearer names, we can skip the `-view` :-) )